### ISSUE TITLE
Location not detected - West Passage

### ISSUE THREAD

**Issue by tatundria** [lud99/botw-unexplored#13]
I have been through the location "West Passage" inside of Hyrule Castle many times, I have even saved the game after visiting that location. However, this is the only location not being marked as completed by botw-unexplored as I now have 186 of 187 total locations. Please, can you help me with this?

**Comment by akouzoukos** [comment]
I have the same issue, but with a different location. The one I'm missing is East Akkala Stable.

**Comment by lud99** [comment]
Just to make sure, have you tried launching the app with the game closed? It can't load the latest savefile when the game is running and instead loads a 'backup', which is then updated when the app is opened while the game is closed.
It's possible that some locations are bugged, so I will check with some of my own savefiles and see what I can do

**Comment by akouzoukos** [comment]
Yeah, I have restarted the game and the app several times. I am on 99.91% with everything else collected and said location already explored, so the locations are probably bugged.

**Comment by akouzoukos** [comment]
Just went and checked the location on my master mode save and the same location appears as undiscovered even though it isn't.
